draft-ietf-capport-architecture.xml

```diff
@@ -496,6 +496,189 @@ o . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . o
+                request.
+            </t>
+        </section>
+        <section anchor="id_recommended_visible_ed" title="Visible to the Enforcement Device">
+            <t>
+                The Enforcement Device will decide on a per packet basis whether it
+                should be permitted to communicate with the external network. Since
+                this decision depends on which User Equipment sent the packet, the
+                Enforcement Device requires that it be able to map the packet to its
+                concept of the User Equipment.
+            </t>
+        </section>
+       </section>
+       <section anchor="id_evaluating" title="Evaluating an Identifier">
+           <t>
+               To evaluate whether an identifer is appropriate, one should consider
+               every recommended property from the perspective of interactions among 
+               the components in the architecture. When comparing identifiers, choose
+               the one which best satifies all of the recommended properties. The
+               architecture does not provide an exact measure of how well an identifier
+               satisfies a given property; care should be taken in performing the
+               evaluation.
+           </t>
+       </section>
+       <section anchor="id_examples" title="Examples of an Identifier">
+           <t>
+               This section provides some examples of identifiers, along with some
+               evaluation of whether they are good identifiers. The list of identifiers
+               is not exhaustive. Other identifiers may be used. An important point to note
+               is that whether the identifiers are good depends heavily on the capabilities
+               of the components and where in the network the components exist.
+           </t>
+           <section anchor="id_example_interface" title="Physical Interface">
+            <t>
+               The physical interface by which the User Equipment is attached to the
+               network can be used to identify the User Equipment. This identifier has
+               the property of being extremely difficult to spoof: the User Equipment is
+               unaware of the property; one User Equipment cannot manipulate its
+               interactions to appear as though it is another.
+           </t>
+
+            <t>
+               Further, if only a single User Equipment is attatched to a given physical
+               interface, then the identifier will be unique. If multiple User Equipment
+               is attached to the network on the same physical interface, then this property
+               is not appropriate.
+           </t>
+
+            <t>
+               Another consideration related to uniqueness of the User Equipment is that
+               if the attached User Equipment changes, both the API server and the
+               Enforcement Device must invalidate their state related to the User Equipment.
```

Comment:
This seems more pertinent for the general section.  (It is a criterion that invalidates the use of a v6 /128.)

Reply:
Something like a requirement that a device have only a single identifier? Or are you thinking something like a requirement that the identifier be easily recycled?

Reply:
The point here is one that Erik raised.  If you identify by a v6 /128, then you prevent the UE from using IPv6 as designed.  Many networks are moving to giving each host its own prefix (see https://tools.ietf.org/html/draft-ietf-v6ops-unique-ipv6-prefix-per-host-01), which allows a number of interesting usage models.  Conditioning captive network access on /128 would prevent those uses.